ballerina-tests/tests/37_streaming_with_deadline_service.bal

```diff
@@ -26,10 +25,9 @@ listener grpc:Listener ep37 = new (9127);
     descMap: getDescriptorMap37StreamingWithDeadline()
 }
 service "HelloWorld37" on ep37 {
-    
+
     remote isolated function callWithDeadline(wrappers:ContextStringStream clientStream) returns wrappers:ContextStringStream|error {
-        log:printInfo("Connected sucessfully.");
-        check clientStream.content.forEach(isolated function(string val) {
+        checkpanic clientStream.content.forEach(isolated function(string val) {
```

Comment:
```suggestion
        check clientStream.content.forEach(isolated function(string val) {
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/2ef479d6607ef5aa5336331221c12c698cdd94cb